todo_app/data/trello_items.py

```diff
@@ -0,0 +1,46 @@
+import requests, os
+from todo_item import Item
+
+api_key = os.getenv("TRELLO_API_KEY")
+board_id = os.getenv("TRELLO_BOARD_ID")
+token = os.getenv("TRELLO_TOKEN")
```

Comment:
Note that for this to work with tests in module 3, you'll need to move these into function calls instead of just having constants at the top of the file.

The issue is that this code currently runs as soon as the file imports happen, before the test has a chance to set up the test environment (including test values for these environment variables)